RISC-V support (#4712)
ComputationalRadiationPhysics/picongpu: include/mpiInfo/CMakeLists.txt

set(CMAKE_CXX_FLAGS_DEFAULT "-Wall")

################################################################################
# Find Threads
################################################################################

# Allow users to override the "-pthread" preference.
if(NOT THREADS_PREFER_PTHREAD_FLAG)
     set(THREADS_PREFER_PTHREAD_FLAG TRUE)
endif()

# WARNING: in case were we cross compile we need to add '-DDMPI_CXX_WORKS=ON` and

> # provide include flags and linker flags manual.

Reviewer: ```suggestion
# provide include flags and linker flags manually.
```

    mpiInfo.cpp
)

target_link_libraries(mpiInfo PRIVATE ${MPIINFO_LIBS} Threads::Threads)
if(RT_LIBRARY AND NOT APPLE)
    # MPI runtime libraries
    target_link_libraries(mpiInfo PRIVATE ${RT_LIBRARY})
endif()
target_link_libraries(mpiInfo PRIVATE MPI::MPI_CXX)

if( CMAKE_TRY_COMPILE_TARGET_TYPE STREQUAL "STATIC_LIBRARY" AND CMAKE_EXE_LINKER_FLAGS)
    # Workaround for linker issues when linking static MPI libraries.

>     # Because of CMAKE_TRY_COMPILE_TARGET_TYPE CMake is providing the statics libraries before the object file from

Reviewer: ```suggestion
    # Because of CMAKE_TRY_COMPILE_TARGET_TYPE CMake providing the statics libraries before the object file from
```

>     # `add_executable` therefore MPI symbols can not be resolved. Linking the linker flaks to the target again will

Reviewer: ```suggestion
    # `add_executable`, MPI symbols can not be resolved. Linking the linker flags to the target again will
```
Is `flaks` a typo?
